agent/bench-scripts/pbench-uperf

```diff
@@ -200,220 +201,220 @@ if [ $? -ne 0 ]; then
 	printf -- "${script_name} $*\n"
 	printf -- "\n"
 	printf -- "\tunrecognized option specified\n\n"
-	usage
+	usage >&2
 	exit 1
 fi
 eval set -- "$opts"
 while true; do
-	case "$1" in
+	arg="${1}"
+	shift
+	case "${arg}" in
 		--install)
-		shift
-		check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
-		exit 0
+			check_install_rpm "${benchmark_rpm}" "${ver}" "${match}"
+			exit 0
 		;;
 		-h|--help)
-		usage
-		exit 0
+			usage
+			exit 0
 		;;
 		--client-label)
-		shift
-		if [ -n "$1" ]; then
-			shift
-		fi
-		warn_log "The --client-label option is deprecated, please use --tool-label-pattern"
+			if [ -n "$1" ]; then
+				shift
+			fi
+			warn_log "The --client-label option is deprecated, please use --tool-label-pattern"
 		;;
```

Comment:
With allowance that you only indented this code, so I won't withhold approval on this basis ...

1. The "deprecated" label seems inappropriate when we're not just "warning that it's going to stop working in the future and recommending that they switch over" ... in fact `--client-label` is **ignored** which, in my mind, is beyond "deprecated". More like "obsolete". E.g., "The --client-label option is obsolete and ignored, please use --tool-label-pattern instead".
2. How long as "deprecated" been in the code? Maybe it's time to just remove this entirely?